packages/next-codemod/bin/upgrade.ts

```diff
@@ -651,3 +661,119 @@ function writeOverridesField(
     }
   }
 }
+
+function warnPeerDependencies(
```

Comment:
This is written in imperative style like it does something. But you're actually just returning something i.e. making a computation. That should be reflected in the function name.